Uplift/Views/Dropdown/DropdownHeaderView.swift

```diff
@@ -13,32 +13,90 @@ protocol DropdownHeaderViewDelegate: class {
 }
 
 class DropdownHeaderView: UIView {
-
+    
+    var filtersAppliedCircle: UIView!
+    var selectedFiltersLabel: UILabel!
+    var titleLabel: UILabel!
```

Comment:
change to `private let titleLabel = UILabel()` and same w/ the other two views. this makes it safer so that we are guaranteed we initialize it and we also prevent anyone from changing it